packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -30,7 +32,9 @@ export default class JournalStore {
     ];
 
     @observable is_filter_dialog_visible = false;
-    @observable unfiltered_messages = [];
+    @observable unfiltered_messages = Array.isArray(localStorage.getObject(storage_key))
```

Comment:
Do we have `getObject` on localStorage? Same for `setObject`.

Reply:
No, we don't. I was using `LocalStorage` from `storage.js` directly and thought this one's methods are the same, will fix it.